src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -233,16 +233,37 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCDriverNew, LinearMethod
   ///whether to do the third part of block lm
   bool block_third;
 
-  //name of the current optimization method, updated by processOptXML before run
+  ///whether to filter parameters for the lm
+  bool filter_param_;
+
+  ///whether to filter parameters for the lm
+  std::string filter_paramStr;
+
+  ///whether to filter parameters for the lm
+  bool filter_info_;
+
+  ///whether to filter parameters for the lm
+  std::string filter_infoStr;
+
+  ///threshold for filtering parameters for the lm
+  double ratio_threshold_;
+
+  ///whether to store samples for the lm
+  bool store_samples_;
+
+  ///whether to store samples for the lm
+  std::string store_samplesStr;
```

Comment:
Could you put all the LMY specific option variables in a structure?

Reply:
I've grouped the optimizer input parameters together in an unnamed struct. I had to leave the string parameters outside the struct to compile successfully, but they are listed just above the struct if this change is just for organizing the file.